src/main/resources/org/jenkinsci/plugins/saml/SamlSecurityRealm/config.jelly

```diff
@@ -27,6 +27,8 @@
         <f:option value="uppercase" selected="${instance.usernameCaseConversion == 'uppercase'}">Uppercase</f:option>
       </select>
     </f:entry>
+
+    <f:dropdownDescriptorSelector field="userIdStrategy" title="${%User name case sensitivity}" default="${descriptor.defaultIdStrategy}"/>
```

Comment:
I think to remove `usernameCaseConversion` can break the instance access unless you don't clean everything inside `<JENKINS_HOME>/users/*`` See that users are created on the instance depending on what is set-up here:
https://github.com/Brantone/saml-plugin/blob/f276d652b7701e7b4ea08401ccc106bba3acf58b/src/main/java/org/jenkinsci/plugins/saml/SamlSecurityRealm.java#L198

I think it is better not to touch this hack for the moment.